theme/Style/Style.qml

```diff
@@ -18,9 +18,6 @@ QtObject {
     property int subLinePixelSize: 10
 
     // Dimensions and sizes
-    property int trayWindowWidth: 400

```

Comment:
I think that this should be kept in the qml code rather than pushing this to c++ side
a better solution would be to add a property in Systray singleton to know the mode: popup or dialog and then have the qml code adjust based on that

Reply:
Ok, but this will introduce conditional logic that is not testable. (I know we don't test anything yet)

Reply:
yes but the fact is that when looking at the qml code you will be able to see everything instead of having to go back and forth between c++ and qml
after all qml is all about showing our stuff to the user

Reply:
okay. done